addons/binding/org.openhab.binding.deconz/README.md

```diff
@@ -40,7 +40,12 @@ Bridge deconz:deconz:homeserver [ host="192.168.0.10" ]
 
 You need to authorize 3rd party applications in deCONZ for the bridge thing to successfully connect. After that an api key to authenticate against the bridge is automatically generated.
 Please note that the generated key cannot be written automatically to the `.thing` file, and has to be set manually. Otherwise, a new api key is generated every time the binding is started.
```

Comment:
At the moment it sounds like you always need to perform this step.
If you just use Paper UI and the auto discovered Things (or manually created Paper UI things), it is not necessary.